### PR TITLE
011-bash-loops [es]

### PR DESCRIPTION
**Pull request by marcelozarate**
## What type of PR is this? (check all applicable)

- [ ] ♻️ Refactor
- [x] ✨ Feature
- [ ] 🐛 Bug Fix
- [ ] 👷 Optimization
- [ ] 📝 Documentation Update
- [ ] 🚩 Other

## Description

Translation of chapter _"Bash Loops"_ to Spanish.

## Added to documentation?

- [ ] 📜 readme
- [ ] 🙅 no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

### CODE REVIEW

File: ebook/es/content/011-bash-loops.md [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+#!/bin/bash
+
+for i in 1 2 3 4 5
+do
+    if [ $i -eq 2 ] 
+    then
+        echo "saltando el número 2"
+        continue
+    fi
+    echo "I es igual a $i"
```
Comment (marcelozarate) [inline ebook/es/content/011-bash-loops.md:143]:
```suggestion
    echo "i es igual a $i"
```

File: ebook/es/content/011-bash-loops.md [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+        then
+            break 2
+        fi
+    echo "bucle interno: $b "
+    done
+done
+```
+
+El script bash comenzará con a=1 y pasará al bucle interno y cuando llegue a b=5, romperá el bucle externo.
+Podemos usar sólo break en lugar de break 2, para romper el bucle interno y ver cómo afecta a la salida.
```
Comment (marcelozarate) [inline ebook/es/content/011-bash-loops.md:198]:
```suggestion
Podemos usar sólo break en lugar de break 2, para romper el bucle interno y ver cómo afecta a la salida.

```

**Review by marcelozarate — COMMENTED** [review]
variable to lowercase and eof blank line

**Review by bobbyiliev — APPROVED** [review]
Thank you!

**Pull request merged**